### ISSUE TITLE
Idea: add the floating marker above the flag in class mode

### ISSUE THREAD

**Issue by 209CATrus** [MT-CTF/capturetheflag#1264]
Add floating (but not compass) marker above the flag in class mpde saying something "PRESS THE CLICK TO SELECT CLASS"

**Comment by farooqkz** [comment]
I agree with this or a mechanism to make switching classes more verbose. I've seen several players who are new and ask how to become scout or medic or change classes.

**Comment by Silvager** [comment]
I like that idea, something possible to add is that the floating marker will only show up if you have never changed classes before and once you have it will disappear, so players who already know how to do it won't be annoyed by the text. Although my idea might be more work to implement than it is worth so IDK

**Comment by LoneWolfHT** [comment]
I think you can check for an empty class meta string for each player, and show the marker based on that
